server/tests/controllers/articles.js

```diff
@@ -142,13 +142,23 @@ describe('Articles controller', () => {
   });
 
   describe('getAllArticle()', () => {
-    it('should return a list of articles', (done) => {
+    it('should return a list of articles, not more than 2 per page', (done) => {
```

Comment:
?? thought this what only when a limit is not given

Reply:
Yes, that's true. I'm changing that now